test/statuses.test.js

```diff
@@ -23,24 +24,9 @@ const datasetsToTest = [
 ]
 
 describe('testing public dataset, in our case finance-vix', function () {
-  it('finance-vix works on production', async function () {
+  it(`finance-vix works on ${DOMAIN}`, async function () {
     this.timeout(180000)
-    const status = await frontendStatus(datasetsToTest[0],process.env.DOMAIN,newLine)
-    expect(status.name).to.equal('finance-vix')
```

Comment:
Why deleting this? this as well was not requested. Merging production and testing test cases?

Reply:
Yes, Instead of duplicating tests for production and testing, I merged them, so using `travis.yml` I can run them separately

Reply:
That's great! just you should have said it in commit message, not here :)